src/Ilios/CoreBundle/DataFixtures/ORM/AbstractMeshFixture.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Ilios\CoreBundle\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\AbstractFixture as DataFixture;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * A data-loader base-class for importing MeSH records from data files.
+ *
+ * Class AbstractMeshFixture
+ * @package Ilios\CoreBundle\DataFixtures\ORM
+ */
+abstract class AbstractMeshFixture extends DataFixture implements
+    FixtureInterface,
+    ContainerAwareInterface
```

Comment:
Extending Symfony\Component\DependencyInjection\ContainerAware means you don't have to implement setContainer.